ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/JiraIntegrationForm.tsx

```diff
@@ -40,6 +40,7 @@ export type JiraIntegration = {
             priorityName: string;
         }[];
         defaultFieldsJson: string;
+        disablePriority: boolean;
```

Comment:
Looking forward to day when we remove this redundant type declaration, can you also in frontend file which corresponds to backend source of truth:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/types/notifier.proto.ts#L106-L113